site/content/dev-guide/custom_http_adapter.md

```diff
@@ -0,0 +1,109 @@
++++
+title = "Implement a Custom Hono HTTP Protocol Adapter"
+weight = 100
```

Comment:
This needs to be a value in the range defined by the menu entries, i.e. in this case something in the range [250, 300). Otherwise the pagination does not reflect the order of the sections anymore.